src/Microsoft.Health.Fhir.Shared.Client/FhirResponse{T}.cs

```diff
@@ -23,5 +23,10 @@ public FhirResponse(HttpResponseMessage response, T resource)
         {
             return response.Resource;
         }
+
+        public T ToT()
```

Comment:
Doesn't the Resource property above already givce you this?

Reply:
Yes, it does. This in response to https://docs.microsoft.com/en-us/visualstudio/code-quality/ca2225?view=vs-2019 which talks about having an alternative member for languages that don't support overloaded operators. I think since this will be a library, it's right to have the method also.